engines/order_management/spec/services/order_management/subscriptions/summary_spec.rb

```diff
@@ -81,6 +90,21 @@ module Subscriptions
           summary.instance_variable_set(:@success_ids, success_ids)
           expect(summary.issue_count).to be 2 # 7 & 9
         end
+
+        context "when there are also subscription issues" do
+          let(:subscription) { double(:subscription, id: 101) }
+          let(:order) { double(:order, id: 1) }
```

Comment:
and the same happens with the order. We must ensure we don't diverge from `Spree::Order`'s public API.